### ISSUE TITLE
metal binary complains about private key

### ISSUE THREAD

**Issue by mwrock** [chef/chef-provisioning#65]
working off of the beta.5 bits, if I run:

```
metal destroy QA1T3NDEVOPSLABCHEF01
```

I receive this error:
```
[2014-05-28T13:43:09-07:00] INFO: Starting chef-zero on port 8889 with repository at repository at /home/mwrock/dev/DevOps/chef-repo
  One version per cookbook

{:chef_server_url=>"http://127.0.0.1:8889", :options=>{:client_name=>"mwrock", :signing_key_filename=>"/home/mwrock/dev/DevOps/chef-repo/.chef/mwrock.pem"}}
/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/mixlib-authentication-1.3.0/lib/mixlib/authentication/signedheaderauth.rb:94:in `private_encrypt': private key needed. (OpenSSL::PKey::RSAError)
```

Note that
```
{:chef_server_url=>"http://127.0.0.1:8889", :options=>{:client_name=>"mwrock", :signing_key_filename=>"/home/mwrock/dev/DevOps/chef-repo/.chef/mwrock.pem"}}
```
is output I hacked in to see the server info.

I'm still figuring my way around chef keys but the above key mwrock.pem is a key that is in my knife.rb and configured for an actual chef server. Should a chef-zero run ignore that key?

Thanks!

**Comment by randomcamel** [comment]
@mwrock Both chef-provisioning and chef-client have come a long way: is this still an issue?

**Comment by mwrock** [comment]
Wow yeah. It has been a while. Just looked at this again and it looks like `metal` is no longer included in the gemspec's executables as of 37ec6ea so its no longer available at the commandline. I'm gonna close this since it appears this has been abandoned.
